source/fuzz/pass_management/repeated_pass_recommender_standard.cpp

```diff
@@ -341,6 +341,12 @@ RepeatedPassRecommenderStandard::GetFuturePassRecommendations(
     // No obvious follow-on passes
     return {};
   }
+
+  if (&pass == pass_instances_->GetSwapFunctions()) {
+    // No obvious follow-on passes
+    return {};
+  }
+
```

Comment:
Again, because this is not a repeated pass you should not deal with it in this file. I.e., there should be no changes to this file.

Reply:
Reverted changes in `repeated_pass_recommender_standard.cpp`